django/cantusdb_project/static/js/uv-loader.js

```diff
@@ -0,0 +1,12 @@
+const urlAdaptor = new UV.IIIFURLAdaptor();
+const manifestFromUrl = urlAdaptor.get("manifest");
+
+if (manifestFromUrl !== undefined)
+{
+    const data = urlAdaptor.getInitialData({
+        manifest: manifestFromUrl,
+    });
+
+    uv = UV.init("uv", data);
+    urlAdaptor.bindTo(uv);
+}
```

Comment:
This is so small, do you really need a separate JS file? Can't you include it in the uv.html file itself?